src/renderer.rs

```diff
@@ -278,7 +280,7 @@ impl Renderer {
     /// Update viewport size.
     pub fn resize(&mut self, size: Size, scale_factor: f64) {
         // Update DPR.
-        let _ = self.entries.set_scale_factor(scale_factor);
+        self.entries.render_at_scale_factor(scale_factor).expect("Rendering icons failed");
```

Comment:
So if resize is never called, the window is not rendered? That doesn't sound optimal.

Reply:
`State::resize()` is always called when a `<State as WindowHandler>::configure()` event is emitted, so that isn’t an issue.  We might rename this method to something like `State::init_or_resize()` or something then.